services/QuillLMS/engines/evidence/app/models/concerns/evidence/research/gen_ai/has_optimal_and_sub_optimal.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Evidence
+  module Research
+    module GenAI
+      module HasOptimalAndSubOptimal
+        extend ActiveSupport::Concern
+
+        OPTIMAL_PREFIXES = [
+          'Nice work',
+          'Good job',
+          'Great job',
+          'Excellent job'
+        ].freeze
+
+        def optimal? = OPTIMAL_PREFIXES.any? { |prefix| text.start_with?(prefix) }
```

Comment:
I think we want to downcase before this comparison on both sides, just to be a little more accurate in case it randomly capitalizes the text, e.g. Nice Work!. We also should strip any leading whitespace from the text before comparison.

Reply:
Good idea on downcasing.